rosetta/object/operation.go

```diff
@@ -33,7 +33,7 @@ import (
 type Operation struct {
 	ID        identifier.Operation `json:"operation_identifier"`
 	Type      string               `json:"type"`
-	Status    string               `json:"status"`
+	Status    string               `json:"status,omitempty"`
```

Comment:
Status and network index - these get populated with empty values when we're returning the operations after parsing the transaction. Both fields are optional, so we should not emit them since we're not explicitly setting them.

Reply:
When it comes to this field, why are we not able to set it? We should be able to deduce the status from the transaction.

Reply:
We can set it as `COMPLETED`. But in that case, the output we emit would not be the same as the input we receive, unless we also require that the input `operations` we receive set this. The field is optional, so I figured it's best to not expect it, **and** to produce the output that is identical to the one we receive.

Reply:
I think I will check this with Lenny from Coinbase.